packages/cqrs/lib/src/cqrs.dart

```diff
@@ -73,6 +73,11 @@ class CQRS {
       try {
         final json = jsonDecode(response.body);
 
+        // Fix for https://github.com/leancodepl/corelibrary/issues/193
```

Comment:
I don't think we need this comment, the reason for this change is well-documented in Git and GH

Reply:
Actually, it's the contracts generator that causes this issue, this is only a slice to fix it. IMHO it's nice to leave it here because it's not our code that is actually responsible for the exception